Cargo.toml

```diff
@@ -20,6 +20,9 @@ travis-ci = { repository = "iliekturtles/uom" }
 coveralls = { repository = "iliekturtles/uom" }
 maintenance = { status = "actively-developed" }
 
+[workspace]
```

Comment:
Something doesn't seem to be correct with the workspace setup. https://doc.rust-lang.org/book/second-edition/ch14-03-cargo-workspaces.html says that running `cargo test` in the root should work for all crates within the workspace. That doesn't seem to be happening, but running `cargo test -p testcrate` does include those tests. If we can get that working then the appveyor and travis config files will only need a single `cargo test` line.

Reply:
I believe this is because of one important distinction missing from that documentation, that is using a [virtual manifest](https://doc.rust-lang.org/cargo/reference/manifest.html#virtual-manifest) at the top-level of the workspace. In the next "Package Selection" section it's noted:

> When `default-members` is not specified, the default is the root manifest if it is a package, or every member manifest (as if `--all` were specified on the command-line) for virtual workspaces.

I could do one of two things:

 1. Add the `testcrate` to the default members so running `cargo test` will test it as well by default.
 2. Switch the CI configuration to use `cargo test --all` to test both crates at once.

I'm leaning towards the first option so that contributors don't _need_ to know that this is a workspace when making small changes, they can just run `cargo test` like normal and all tests will be run.